src/index.js

```diff
@@ -240,18 +213,18 @@ class Dropzone extends React.Component {
     this.node = ref;
   }
 
-  fileAccepted(file, props = this.props) {
+  fileAccepted(file) {
```

Comment:
Could this function be a pure one? Let's remove the dependency on `this` here and extract it from the class. There is no reason for it to be a class instance method.